src/components/Toaster/types.ts

```diff
@@ -0,0 +1,26 @@
+export type ToasterArgs = {
+    // FIXME: BREAKING CHANGE. Rename to "className"
+    additionalClass?: string;
```

Comment:
We can add `className` prop as alias for `additionalClass ` and deprecate this with JSDoc